sasha-shevchenko/eyeq-table
# Table overview page

*** New Feature: Table overview in restaurant dashboard ***

- Created route for dashboards#overview & updated current route to dashboards#show
- Updated dashboards_controller with overview method. Disclaimer: work in progress, as with this current logic we are only able to display the order of Session.last - however works fine for planned demo of this NVP version. 
- Created the view for dashboards#overview (styling is still in progress)
- Updated Session model with 'lap-solution' method for displaying ordered quantity of session_items on the dashboards#overview.
-  Updated restaurant navbar with a link to the dashboards#overview
- Updated all redirect_to with new link path in categories_controller, so nothing will be breaking inside dashboards#show. 


**New route paths for restaurant dashboard**

Dashboards#show - restaurants/:id/dashboard
Dashboards#overview - restaurants/:id/overview


**Testing**

To test this branch you need to:

1. Login as a restaurant and test all actions in restaurants/:id/dashboard
2. Create a new session -> send items to kitchen -> check table overview page for order.
3. Add additional items to current order -> check table overview page for updated order.
4. Request bill for current order -> check table overview page.
